www/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -19,20 +19,23 @@ import {StickyContainer} from 'react-sticky';
 import PropTypes from 'prop-types';
 import React from 'react';
 import StickySidebar from 'components/StickySidebar';
+import TitleAndMetaTags from 'components/TitleAndMetaTags';
 import findSectionForPath from 'utils/findSectionForPath';
 import toCommaSeparatedList from 'utils/toCommaSeparatedList';
 import {sharedStyles} from 'theme';
-
-// TODO Use 'react-helmet' to set metadata
+import {urlRoot} from 'constants';
 
 const MarkdownPage = ({
   authors,
   date,
+  ogDescription,
   location,
   markdownRemark,
   sectionList,
+  titlePostfix,
```

Comment:
(At least until we have Flow covering us) maybe we should initialize `titlePostfix` to an empty string (to avoid potentially adding "undefined") to the generated title below?

Reply:
+1 will fix